<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>481</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>定时器</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="radioContinue">
       <property name="text">
        <string>连续定时</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>定时周期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="btnDynamic">
       <property name="text">
        <string>动态创建单次定时器</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QRadioButton" name="radioSingle">
       <property name="text">
        <string>单次定时</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>2000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>当前时间（时分秒）</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdhour">
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdmin">
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdsec">
       <property name="digitCount">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>定时器精度</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioPrecise">
       <property name="text">
        <string>Precise Timer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioCoarse">
       <property name="text">
        <string>Coarse Timer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioVeryCoarse">
       <property name="text">
        <string>VeryCoarse Timer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labElapsed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
